Resources/doc/reference/configuration/headers.rst

```diff
@@ -218,12 +249,22 @@ This value must be a valid input to ``DateTime``.
                     headers:
                         last_modified: "-1 hour"
 
-.. hint::
+.. note::
 
     Setting an arbitrary last modified time allows clients to send
     ``If-Modified-Since`` requests. Varnish can handle these to serve data
     from the cache if it was not invalidated since the client requested it.
 
+    If you only use these simple configurations to add cache validation
+    information, there is no point in adding both a last modified and ETag
+    header. Usually, you should prefer the ETag method as it is not prone to
+    clock skew issues.
```

Comment:
This is slightly tendentious. There is [RFC 2616 14.25](http://tools.ietf.org/html/rfc2616#section-14.25) which discusses clock skew-related issues with `If-Modified-Since`, but [13.3.4 ‘Rules for When to Use Entity Tags and Last-Modified Dates’](http://tools.ietf.org/html/rfc2616#section-13.3.4) is more nuanced. Perhaps we should at least link to the relevant parts in the RFC so we invite readers to form an opinion of their own.